src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  padding: 100px;
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    width: $width-blocks-calendar;
+    gap: $shift;
+    padding: $isnside-shift;
+
+    @for $index from 1 through $max-days-of-month {
+      & div:nth-child(#{$index}) {
```

Comment:
add class for div and use class

Reply:
also, you can styles for class outside cycle (you need to add these styles for all days)
on cycle should be just it 👇

```
::before {
  content: "#{$index}";
}
```